Makefile

```diff
@@ -1,10 +1,2 @@
 build:
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o main ./main.go
-docker_image:
```

Comment:
I believe it's useless, not sure about build target, but if it's also useless, we can remove the whole Makefile